jobs/sql/extractions/salud_mental_encuentro_staging.sql

```diff
@@ -1,1025 +1,331 @@
+where o.obs_group_order = 5;
+
+update salud_mental_encountero e
+    inner join temp_mh_medication_name_obs o on e.encounter_id = o.encounter_id
+set e.medication_6_obs_group_id = o.obs_group_id, e.medication_6_name = drugName(o.value_drug)
+where o.obs_group_order = 6;
+
+update salud_mental_encountero e
+    inner join temp_mh_medication_name_obs o on e.encounter_id = o.encounter_id
+set e.medication_7_obs_group_id = o.obs_group_id, e.medication_7_name = drugName(o.value_drug)
+where o.obs_group_order = 7;
+
+update salud_mental_encountero e
+    inner join temp_mh_medication_name_obs o on e.encounter_id = o.encounter_id
+set e.medication_8_obs_group_id = o.obs_group_id, e.medication_8_name = drugName(o.value_drug)
+where o.obs_group_order = 8;
+
+update salud_mental_encountero e
+    inner join temp_mh_medication_name_obs o on e.encounter_id = o.encounter_id
+set e.medication_9_obs_group_id = o.obs_group_id, e.medication_9_name = drugName(o.value_drug)
+where o.obs_group_order = 9;
+
+update salud_mental_encountero e
+    inner join temp_mh_medication_name_obs o on e.encounter_id = o.encounter_id
+set e.medication_10_obs_group_id = o.obs_group_id, e.medication_10_name = drugName(o.value_drug)
+where o.obs_group_order = 10;
+
+-- Once the medication names are established with the relevent groups, other group members can be added
+
+update salud_mental_encountero set medication_1_instructions = obs_from_group_id_value_text_from_temp(medication_1_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_2_instructions = obs_from_group_id_value_text_from_temp(medication_2_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_3_instructions = obs_from_group_id_value_text_from_temp(medication_3_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_4_instructions = obs_from_group_id_value_text_from_temp(medication_4_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_5_instructions = obs_from_group_id_value_text_from_temp(medication_5_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_6_instructions = obs_from_group_id_value_text_from_temp(medication_6_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_7_instructions = obs_from_group_id_value_text_from_temp(medication_7_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_8_instructions = obs_from_group_id_value_text_from_temp(medication_8_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_9_instructions = obs_from_group_id_value_text_from_temp(medication_9_obs_group_id, 'PIH', '9072');
+update salud_mental_encountero set medication_10_instructions = obs_from_group_id_value_text_from_temp(medication_10_obs_group_id, 'PIH', '9072');
+
+update salud_mental_encountero set medication_1_units = obs_from_group_id_value_text_from_temp(medication_1_obs_group_id, 'PIH', '9071');
```

Comment:
This is numeric so value_text will not work here, we need to add a new function to return the value_numeric from temp, i added this new function to existing functions obs_from_group_id_value_numeric_from_temp

Reply:
OK